README.rst

```diff
@@ -30,7 +30,92 @@ dashboard, please contact your Optimizely account executive.
 Using the SDK
 ~~~~~~~~~~~~~
 
-See the Optimizely `Full Stack documentation`_ to learn how to
+You can initialize the Optimizely instance in three ways: with a datafile, by providing an `sdk_key`, or by providing an implementation of `config_manager.BaseConfigManager`_. Each method is described below.
+
+1. Initialize Optimizely with a datafile. This datafile will be used as
+   ProjectConfig throughout the life of Optimizely instance.
+   ::
+
+      optimizely.Optimizely(
+        datafile
+      )
+
+2. Initialize Optimizely by providing an 'sdk_key'. This will initialize
+   a PollingConfigManager that makes an HTTP GET request to the URL (formed 
+    using your provided `sdk key` and the default datafile CDN URL
+   template) to asynchronously download the project datafile at regular
+   intervals and update ProjectConfig when a new datafile is recieved. A
+   hard-coded datafile can also be provided along with the `sdk_key` that
+   will be used initially before any update.
+   ::
+
+      optimizely.Optimizely(
```

Comment:
since you mentioned the hard-coded datafile, would it make sense to put in this code sample or have another code sample below this that uses a hard-coded one too

Reply:
Good point. Let me add that example.